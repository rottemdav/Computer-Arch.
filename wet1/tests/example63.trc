1 1 20 1 local_history local_tables using_share_lsb
0x950 T 0x128d204
0xd38 T 0xbae5b8
0xf67bf98 T 0xbfa77024
0x29c536c N 0x1e8c
0xc34 T 0x49528
0x9d1c50 T 0x240fd8
0x7708 T 0x7b80fc90
0x1d3d214c N 0xe7574
0x0a3788 T 0x58e4
0x340 T 0xac691da4
0xb414c40 N 0x71d7f480
0x513d7c T 0x278c8
0xd38 N 0xbae5b8
0x8689c20 T 0xc22a885c
0xdd018 T 0x5da26c
0x1d3d214c N 0xe7574
0xb414c40 N 0x71d7f480
0x0a3788 N 0x58e4
0x340 T 0xac691da4
0x4799715c T 0xd80
0x0a3788 N 0x58e4
0x8689c20 N 0xc22a885c
0x513d7c N 0x278c8
0xc34 N 0x49528
0xc34 T 0x49528
0xf67bf98 N 0xbfa77024
0x8689c20 N 0xc22a885c
0xe900 N 0x83c
0x75eac N 0x8f348
0x60e00580 N 0x587ea80
0x950 T 0x128d204
0xf56edc44 N 0xdedc
0x8689c20 T 0xc22a885c
0x0a3788 N 0x58e4
0x950 N 0x128d204
0xd38 T 0xbae5b8
0x29c536c T 0x1e8c
0x950 N 0x128d204
0x06eb0898 T 0x96b9448
0x5ef8 N 0x1d7cf0
0x85f11898 N 0xc17c
0x75eac T 0x8f348
0x340 T 0xac691da4
0x1d3d214c T 0xe7574
0xf67bf98 N 0xbfa77024
0x60e00580 N 0x587ea80
0x51c063c N 0x8fd54
0x85f11898 T 0xc17c
0x950 N 0x128d204
0x950 T 0x128d204
0xd38 T 0xbae5b8
0x4799715c N 0xd80
0xef00 N 0x4ab944
0x513d7c T 0x278c8
0xb414c40 T 0x71d7f480
0x4799715c N 0xd80
0xb414c40 T 0x71d7f480
0x4799715c N 0xd80
0x340 T 0xac691da4
0x711d1f8 T 0x89e8
0x0a3788 T 0x58e4
0xc70 T 0x26ec3f00
0x8689c20 T 0xc22a885c
0x711d1f8 N 0x89e8
0xb414c40 N 0x71d7f480
0x7708 T 0x7b80fc90
0x85f11898 T 0xc17c
0x8689c20 T 0xc22a885c
0x4799715c T 0xd80
0x950 T 0x128d204
0xf67bf98 T 0xbfa77024
0xc34 N 0x49528
0x06eb0898 N 0x96b9448
0xef00 T 0x4ab944
0x75eac T 0x8f348
0x0a3788 N 0x58e4
0x9d1c50 N 0x240fd8
0x1d3d214c T 0xe7574
0xc34 N 0x49528
0x8689c20 T 0xc22a885c
0x4799715c T 0xd80
0x450338 T 0x72da268
0x60e00580 T 0x587ea80
0x9d1c50 T 0x240fd8
0x9d1c50 T 0x240fd8
0xe900 T 0x83c
0xe900 N 0x83c
0x9d1c50 N 0x240fd8
0x75eac T 0x8f348
0xef00 T 0x4ab944
0xef00 N 0x4ab944
0x9d1c50 T 0x240fd8
0x450338 N 0x72da268
0xc70 T 0x26ec3f00
0x1d3d214c T 0xe7574
0x29c536c T 0x1e8c
0x8689c20 N 0xc22a885c
0x8689c20 T 0xc22a885c
0x513d7c T 0x278c8
0x950 T 0x128d204
0xef00 N 0x4ab944
0x513d7c T 0x278c8
0xb414c40 N 0x71d7f480
0x4799715c T 0xd80
0x85f11898 T 0xc17c
0x8689c20 T 0xc22a885c
0xc70 N 0x26ec3f00
0xf56edc44 N 0xdedc
0x1d3d214c N 0xe7574
0x85f11898 N 0xc17c
0xc34 T 0x49528
0x7708 T 0x7b80fc90
0xd38 N 0xbae5b8
0x450338 N 0x72da268
0x5ef8 T 0x1d7cf0
0x29c536c N 0x1e8c
0x0a3788 T 0x58e4
0xe900 N 0x83c
0xd38 N 0xbae5b8
0x950 T 0x128d204
0x29c536c T 0x1e8c
0x75eac T 0x8f348
0xd38 N 0xbae5b8
0x7708 N 0x7b80fc90
0x711d1f8 N 0x89e8
0x29c536c T 0x1e8c
0x061c98 N 0x2037932c
0xc70 T 0x26ec3f00
0x06eb0898 T 0x96b9448
0xf67bf98 N 0xbfa77024
0x5ef8 N 0x1d7cf0
0x711d1f8 T 0x89e8
0x450338 T 0x72da268
0x06eb0898 N 0x96b9448
0x450338 N 0x72da268
0x450338 N 0x72da268
0x4799715c T 0xd80
0xf56edc44 N 0xdedc
0x60e00580 N 0x587ea80
0xd38 N 0xbae5b8
0x9d1c50 N 0x240fd8
0x5ef8 T 0x1d7cf0
0x51c063c N 0x8fd54
0x8689c20 T 0xc22a885c
0x85f11898 T 0xc17c
0x450338 N 0x72da268
0x9d1c50 N 0x240fd8
0x5ef8 N 0x1d7cf0
0xef00 N 0x4ab944
0x85f11898 T 0xc17c
0xdd018 N 0x5da26c
0x950 N 0x128d204
0xc70 N 0x26ec3f00
0x51c063c T 0x8fd54
0x7708 T 0x7b80fc90
0xf56edc44 N 0xdedc
0xc70 N 0x26ec3f00
0x513d7c N 0x278c8
0x950 T 0x128d204
0xc70 N 0x26ec3f00
0xd38 T 0xbae5b8
0x711d1f8 T 0x89e8
0xc34 N 0x49528
0x450338 N 0x72da268
0x1d3d214c T 0xe7574
0xb414c40 N 0x71d7f480
0xd38 N 0xbae5b8
0x75eac N 0x8f348
0xe900 T 0x83c
0x1d3d214c T 0xe7574
0x75eac T 0x8f348
0x51c063c T 0x8fd54
0x0a3788 N 0x58e4
0xd38 T 0xbae5b8
0x8689c20 N 0xc22a885c
0x061c98 N 0x2037932c
0x51c063c N 0x8fd54
0x450338 T 0x72da268
0x7708 N 0x7b80fc90
0x711d1f8 N 0x89e8
0xdd018 T 0x5da26c
0xc34 T 0x49528
0xef00 N 0x4ab944
0x711d1f8 N 0x89e8
0x5ef8 T 0x1d7cf0
0xdd018 T 0x5da26c
0xe900 T 0x83c
0x061c98 T 0x2037932c
0x51c063c N 0x8fd54
0x340 T 0xac691da4
0x5ef8 T 0x1d7cf0
0x711d1f8 N 0x89e8
0x950 T 0x128d204