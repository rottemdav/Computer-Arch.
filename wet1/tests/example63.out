0x950 N 0x954
0xd38 N 0xd3c
0xf67bf98 N 0xf67bf9c
0x29c536c N 0x29c5370
0xc34 N 0xc38
0x9d1c50 N 0x9d1c54
0x7708 N 0x770c
0x1d3d214c N 0x1d3d2150
0xa3788 N 0xa378c
0x340 N 0x344
0xb414c40 N 0xb414c44
0x513d7c N 0x513d80
0xd38 N 0xd3c
0x8689c20 N 0x8689c24
0xdd018 N 0xdd01c
0x1d3d214c N 0x1d3d2150
0xb414c40 N 0xb414c44
0xa3788 N 0xa378c
0x340 N 0x344
0x4799715c N 0x47997160
0xa3788 N 0xa378c
0x8689c20 N 0x8689c24
0x513d7c N 0x513d80
0xc34 N 0xc38
0xc34 N 0xc38
0xf67bf98 N 0xf67bf9c
0x8689c20 N 0x8689c24
0xe900 N 0xe904
0x75eac N 0x75eb0
0x60e00580 N 0x60e00584
0x950 N 0x954
0xf56edc44 N 0xf56edc48
0x8689c20 N 0x8689c24
0xa3788 N 0xa378c
0x950 N 0x954
0xd38 N 0xd3c
0x29c536c N 0x29c5370
0x950 N 0x954
0x6eb0898 N 0x6eb089c
0x5ef8 N 0x5efc
0x85f11898 N 0x85f1189c
0x75eac N 0x75eb0
0x340 N 0x344
0x1d3d214c N 0x1d3d2150
0xf67bf98 N 0xf67bf9c
0x60e00580 N 0x60e00584
0x51c063c N 0x51c0640
0x85f11898 N 0x85f1189c
0x950 N 0x954
0x950 N 0x954
0xd38 N 0xd3c
0x4799715c N 0x47997160
0xef00 N 0xef04
0x513d7c N 0x513d80
0xb414c40 N 0xb414c44
0x4799715c N 0x47997160
0xb414c40 N 0xb414c44
0x4799715c N 0x47997160
0x340 N 0x344
0x711d1f8 N 0x711d1fc
0xa3788 N 0xa378c
0xc70 N 0xc74
0x8689c20 N 0x8689c24
0x711d1f8 N 0x711d1fc
0xb414c40 N 0xb414c44
0x7708 N 0x770c
0x85f11898 N 0x85f1189c
0x8689c20 N 0x8689c24
0x4799715c N 0x47997160
0x950 N 0x954
0xf67bf98 N 0xf67bf9c
0xc34 N 0xc38
0x6eb0898 N 0x6eb089c
0xef00 N 0xef04
0x75eac N 0x75eb0
0xa3788 N 0xa378c
0x9d1c50 N 0x9d1c54
0x1d3d214c N 0x1d3d2150
0xc34 N 0xc38
0x8689c20 N 0x8689c24
0x4799715c N 0x47997160
0x450338 N 0x45033c
0x60e00580 N 0x60e00584
0x9d1c50 N 0x9d1c54
0x9d1c50 N 0x9d1c54
0xe900 N 0xe904
0xe900 N 0xe904
0x9d1c50 N 0x9d1c54
0x75eac N 0x75eb0
0xef00 N 0xef04
0xef00 N 0xef04
0x9d1c50 N 0x9d1c54
0x450338 N 0x45033c
0xc70 N 0xc74
0x1d3d214c N 0x1d3d2150
0x29c536c N 0x29c5370
0x8689c20 N 0x8689c24
0x8689c20 N 0x8689c24
0x513d7c N 0x513d80
0x950 N 0x954
0xef00 N 0xef04
0x513d7c N 0x513d80
0xb414c40 N 0xb414c44
0x4799715c N 0x47997160
0x85f11898 N 0x85f1189c
0x8689c20 N 0x8689c24
0xc70 N 0xc74
0xf56edc44 N 0xf56edc48
0x1d3d214c N 0x1d3d2150
0x85f11898 N 0x85f1189c
0xc34 N 0xc38
0x7708 N 0x770c
0xd38 N 0xd3c
0x450338 N 0x45033c
0x5ef8 N 0x5efc
0x29c536c N 0x29c5370
0xa3788 N 0xa378c
0xe900 N 0xe904
0xd38 N 0xd3c
0x950 N 0x954
0x29c536c N 0x29c5370
0x75eac N 0x75eb0
0xd38 N 0xd3c
0x7708 N 0x770c
0x711d1f8 N 0x711d1fc
0x29c536c N 0x29c5370
0x61c98 N 0x61c9c
0xc70 N 0xc74
0x6eb0898 N 0x6eb089c
0xf67bf98 N 0xf67bf9c
0x5ef8 N 0x5efc
0x711d1f8 N 0x711d1fc
0x450338 N 0x45033c
0x6eb0898 N 0x6eb089c
0x450338 N 0x45033c
0x450338 N 0x45033c
0x4799715c N 0x47997160
0xf56edc44 N 0xf56edc48
0x60e00580 N 0x60e00584
0xd38 N 0xd3c
0x9d1c50 N 0x9d1c54
0x5ef8 N 0x5efc
0x51c063c N 0x51c0640
0x8689c20 N 0x8689c24
0x85f11898 N 0x85f1189c
0x450338 N 0x45033c
0x9d1c50 N 0x9d1c54
0x5ef8 N 0x5efc
0xef00 N 0xef04
0x85f11898 N 0x85f1189c
0xdd018 N 0xdd01c
0x950 N 0x954
0xc70 N 0xc74
0x51c063c N 0x51c0640
0x7708 N 0x770c
0xf56edc44 N 0xf56edc48
0xc70 N 0xc74
0x513d7c N 0x513d80
0x950 N 0x954
0xc70 N 0xc74
0xd38 N 0xd3c
0x711d1f8 N 0x711d1fc
0xc34 N 0xc38
0x450338 N 0x45033c
0x1d3d214c N 0x1d3d2150
0xb414c40 N 0xb414c44
0xd38 N 0xd3c
0x75eac N 0x75eb0
0xe900 N 0xe904
0x1d3d214c N 0x1d3d2150
0x75eac N 0x75eb0
0x51c063c N 0x51c0640
0xa3788 N 0xa378c
0xd38 N 0xd3c
0x8689c20 N 0x8689c24
0x61c98 N 0x61c9c
0x51c063c N 0x51c0640
0x450338 N 0x45033c
0x7708 N 0x770c
0x711d1f8 N 0x711d1fc
0xdd018 N 0xdd01c
0xc34 N 0xc38
0xef00 N 0xef04
0x711d1f8 N 0x711d1fc
0x5ef8 N 0x5efc
0xdd018 N 0xdd01c
0xe900 N 0xe904
0x61c98 N 0x61c9c
0x51c063c N 0x51c0640
0x340 N 0x344
0x5ef8 N 0x5efc
0x711d1f8 N 0x711d1fc
0x950 N 0x954
flush_num: 100, br_num: 193, size: 56b
